concepticondata/conceptlists/Verheyen-2019-1000.tsv

```diff
@@ -0,0 +1,1001 @@
+Verheyen-2019-1000-147	147	deftig	distinguished			7.36842105263158	3.7	4.45	4.6	4.7	6	4.85	6	2	35143	2	647	153	2.45425437024628	0.000384650880237
+Verheyen-2019-1000-148	148	democratisch	democratic			11.6	4.55	5.25	4.25	4.35	4.7	5.05	12	4	86368	0	1553	162	0.095882771835186	2.33520136425744E-05
+Verheyen-2019-1000-149	149	depressief	depressed			9.85	2.4	4.55	4.2	5.1	4.45	1.5	10	3	67938	2	246	518	1.71757859227863	0.000262098367168
+Verheyen-2019-1000-150	150	derde	three			5.05	3.6	5.31578947368421	3.25	4.7	4.5	3.4	5	2	71938	1	0	1	0.471476297335574	0.000108351260625
+Verheyen-2019-1000-151	151	deugdzaam	good	GOOD	1035	11.1	3.65	4.05263157894737	4.15	3.2	3.8	5.55	9	2	39406	0	85	53	0.074234850877636	3.6554232057837E-05
+Verheyen-2019-1000-152	152	dicht	close			4.7	3	4.85	4	4.75	4.6	3.3	5	1	29429	6	8733	8103	3.14656154274427	0.000472677601674
+Verheyen-2019-1000-153	153	dichtbij	near	NEAR	1942	5.55	4	4.2	4	5.55	5.5	4.85	8	2	43558	0	2273	2162	2.59100803837547	0.000438201707509
+Verheyen-2019-1000-154	154	dichter	closer			6.2	3.8	4.3	3.85	5.75	4.05	4.4	7	2	89215	5	8733	948	1.22155953467085	0.000218550896911
+Verheyen-2019-1000-155	155	diep	deep	DEEP	1593	5.25	4.05	3.75	4.45	5.75	5.5	3.65	4	1	26033	8	11667	4407	2.81666136493418	0.000819019160757
+Verheyen-2019-1000-156	156	dieper	lower			5.85	4.05	4.3	3.8	5.05	5.25	4.25	6	2	73249	5	11667	0	0.06139941220312	1.82429471126355E-05
+Verheyen-2019-1000-157	157	diepzinnig	deep			11.75	3.45	4.75	4.45	4.4	3.45	5.2	10	3	61860	0	136	82	0.17716823693768	4.24703358485561E-05
+Verheyen-2019-1000-158	158	dierbaar	dear	DEAR	2470	9.1	4.8	4.75	4.4	3.7	4.65	6.05	8	2	95572	0	378	644	0.276330734193963	5.12208325995152E-05
+Verheyen-2019-1000-159	159	dik	thick	THICK	1244	4.6	3	5.6	3.3	5.75	6.5	2.35	3	1	7928	22	6689	3898	15.8975430532018	0.00194442611255
+Verheyen-2019-1000-160	160	direct	direct			6.25	5.1	4.85	5.05	5.8	3.35	4.7	6	2	25925	0	7155	3381	3.76557796373354	0.000406860675821
+Verheyen-2019-1000-161	161	doeltreffend	effective			10.95	5.3	4.36842105263158	5.4	3.75	5	5.3	12	3	103409	0	495	55	NaN	NaN
+Verheyen-2019-1000-162	162	dof	pale			9.25	2.21052631578947	3.95	2.6	2.3	3.45	2.9	3	1	4474	18	768	49	0.633876874828853	0.000174619433119
+Verheyen-2019-1000-163	163	dol	mad	MAD	1899	7	6	3.25	4.47368421052632	3.7	4.05	5.1	3	1	7983	26	1059	2594	0.187017491186001	0.000114205534948
+Verheyen-2019-1000-164	164	dom	ignorant	BE IGNORANT	2651	5.85	2.95	3.95	2.6	5.55	5.15	2.35	3	1	7030	19	2096	5240	12.6150889861829	0.001863995737802
+Verheyen-2019-1000-165	165	dominant	dominant			10.75	5.5	4.75	6.15	4.3	5.85	3.2	8	3	56622	0	289	113	1.06532303687319	0.000199328440865
+Verheyen-2019-1000-166	166	donker	dismally			4.3	2.7	5.25	4.75	5.15	6.3	2.8	6	2	65866	7	7403	3814	18.5348765867472	0.001867357308328
+Verheyen-2019-1000-167	167	dood	death	DEAD	1625	5.15	1.95	6.2	4.35	5.1	5.75	1.3	4	1	15241	12	6431	40286	26.4497342633645	0.003120052693459
```

Comment:
the concept is correctly mapped, but the translation should be dead not death, given that it is an adj list